docs/docs/adrs/adr-008-throttle-retries.md

```diff
@@ -67,46 +89,57 @@ Two things are achieved with this approach:
+If the consumer is following the queuing/blocking protocol described, then no bad behavior occurs and the _VSC Maturity and Slashing Order_ property is maintained.
 
-If a consumer blocks the sending of VSCMatured packets: The consumer is malicious and blocking vsc matured packets that should have been sent. This will block unbonding only up until the VSC timeout period has elapsed. At that time, the consumer is removed. Again the malicious behavior only creates a negative outcome for the chain that is being malicious.
+If a consumer sends `VSCMaturedPackets` too leniently -- the consumer is malicious and sends duplicate `VSCMaturedPackets`, or sends the packets sooner than the CCV protocol specifies -- then the provider needs to handle `VSCMaturedPackets` immediately to prevent DOS, state bloat, or other issues. 
+The only possible negative outcome is that the malicious consumer may not be able to jail a validator who should have been jailed. 
+The malicious behavior only creates a negative outcome for the consumer chain that is being malicious.
+
+If a consumer blocks the sending of `VSCMaturedPackets`, then unbonding operations on the provider will be delayed, but only until the VSC timeout period has elapsed. 
+At that time, the consumer is removed. 
+Again the malicious behavior only creates a negative outcome for the consumer chain that is being malicious.
 
 ### Splitting of PRs and Upgrade Order
 
-This feature will implement consumer changes in [#1024](https://github.com/cosmos/interchain-security/pull/1024). Note these changes should be deployed to prod for all consumers before the provider changes are deployed to prod. That is the consumer changes in #1024 are compatible with the current ("v1") provider implementation of throttling that's running on the Cosmos Hub as of July 2023.
+This feature will implement consumer changes in [#1024](https://github.com/cosmos/interchain-security/pull/1024). 
+
+❗***These changes should be deployed to production for all consumers before the provider changes are deployed to production.*** 
+
+In other words, the consumer changes in [#1024](https://github.com/cosmos/interchain-security/pull/1024) are compatible with the current ("v1") provider implementation of throttling that's running on the Cosmos Hub as of July 2023.
 
-Once all consumers have deployed the changes in #1024, the provider changes from (TBD) can be deployed to prod, fully enabling v2 throttling.
+Once all consumers have deployed the changes in #1024, the provider changes from [#1321](https://github.com/cosmos/interchain-security/pull/1321) can be deployed to production, fully enabling v2 throttling.
```

Comment:
Not sure this is feasible. The provider schedule should not depend on the consumers or vice versa.

Reply:
Ideally it should not. But it's hard to modify existing logic and add new logic that affect both provider and consumer without needing some coordination.